Uplift/Views/Cells/PickupGameCell.swift

```diff
@@ -0,0 +1,146 @@
+//
+//  PickupGameCell.swift
+//  Uplift
+//
+//  Created by Artesia Ko on 2/19/20.
+//  Copyright © 2020 Cornell AppDev. All rights reserved.
+//
+
+import SnapKit
+import UIKit
+
+class PickupGameCell: UICollectionViewCell {
+    // MARK: - Labels
+    private let locationLabel = UILabel()
+    private let playersLabel = UILabel()
+    private let titleLabel = UILabel()
+    private let detailLabel = UILabel()
+    
+    // MARK: - Other Views
+    private let statusButton = UIButton()
+    private let playerIcon = UIImageView()
+    
+    // MARK: - Data
+    private var status: GameStatus = .open
+    
+    // TODO: add delegate for toggling game status
+    
+    enum GameStatus {
```

Comment:
Can we make this a private enum?

Reply:
We need to use it in the configure function, ie. need to access from view controller itself when we dequeue cells, so no.

Reply:
actually nvm I'm going to move the enum to models